src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -359,11 +359,7 @@ def SetThreadOperationalDataset(self, threadOperationalDataset):
     def ResolveNode(self, nodeid):
         self.CheckIsActive()
 
-        try:
-            self.GetConnectedDeviceSync(nodeid, allowPASE=False)
-            return 0
-        except ChipStackError as ex:
-            return ex.err
+        self.GetConnectedDeviceSync(nodeid, allowPASE=False)
```

Comment:
ResolveNode callers check its return value, no?

Reply:
Yes, this changes the `ResolveNode` API, I hope that is OK. This is a more Pythonic way of error handling, especially since it is internally already an exception.

Hm, there are some uses of `ResolveNode` indeed, 3 in total from what I can tell. It seems that `src/controller/python/test/test_scripts/base.py` does no checking, so this change at least will lead to a stack trace instead of just silently failing. I've addressed the other two uses (`src/controller/python/chip-device-ctrl.py` and `src/test_driver/mbed/integration_tests/common/utils.py`).

Reply:
Right, changing the API is fine; I was asking for the callers to be updated.